apps/transaction/templates/transaction/list.html

```
@@ -161,153 +161,153 @@
+    <div class="col-12 col-lg-8">
+      <div class="d-flex align-items-md-start justify-content-between gap-3 mb-3">
+        <div>
+          <h4 class="mb-1">{% trans "Room transactions" %}</h4>
+          {% if transactions_available %}
+            <p class="text-muted small mb-0">{% trans "Click a row to explore the full breakdown." %}</p>
+          {% else %}
+            <p class="text-muted small mb-0">{% trans "Record your first expense to see the running history here." %}</p>
+          {% endif %}
+        </div>
+        <div class="d-flex align-items-center justify-content-end gap-2">
+          <button type="button"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)